extensions/wikia/MercuryApi/handlers/MercuryApiMainPageHandler.class.php

```diff
@@ -39,21 +45,20 @@ public static function getMainPageData( MercuryApi $mercuryApiModel ) {
 		return $mainPageData;
 	}
 
-	public static function getCuratedContentData( MercuryApi $mercuryApiModel, $section = null ) {
+	public static function getCuratedContentData( MercuryApi $mercuryApiModel, $section = null, $newFormat=false ) {
 		$data = [ ];
-
 		try {
 			$data = WikiaDataAccess::cache(
-				self::curatedContentDataMemcKey( $section ),
+				self::curatedContentDataMemcKey( $section . $newFormat ),
```

Comment:
Because of this, we have to wait `WikiaResponse::CACHE_STANDARD` before doing the clean up, right? Not that it's bad, just saying.